### ISSUE TITLE
Publish sample project to Azure and get 404

### ISSUE THREAD

**Issue by lydart** [SoftwareAteliers/asp-net-core-vue-starter#23]
It works fine while running locally, but server returns 404 while deplyed to Azure cloud.
Any configuration need to be updated?

Environment:
- DotNet Core 3.0

**Comment by liborpansky** [comment]
Hi, I tested previous version on .NET Core 2.2 and it worked without problems. As soon as I get home I will test it.

**Comment by gtbee** [comment]
I have same issue. 404 error after deployment to a new Azure app service.

When I created a new project based off the Visual Studio ASP.Net Core 3.0 React template and deployed it to the same App Service it worked without a problem.

**Comment by liborpansky** [comment]
Hi, after several hours of debug and strange Azure exceptions being returned like 
`System.InvalidOperationException:
System.IO.FileNotFoundException:
   at System.Reflection.RuntimeAssembly.nLoad (System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName (System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Reflection.Assembly.Load (System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Reflection.Assembly.Load (System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.AspNetCore.Hosting.GenericWebHostBuilder.ExecuteHostingStartups (Microsoft.AspNetCore.Hosting, Version=3.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60)`

I realized that the template was missing UseSpa extension to load client app in Production environment. Bug has been fixed and now I'm releasing a new version of template.

Cheers!
